src/drive/containers/File.jsx

```diff
@@ -41,7 +42,8 @@ export const getTypeFromMimeType = (collection, prefix = '') => mimetype => {
     return type
   }
   if (type === 'application') {
-    return mappingMimetypeSubtype[subtype]
+    const key = subtype.match(Object.keys(mappingMimetypeSubtype).join('|'))[0]
```

Comment:
This is were a comment would be good :). The code is not explicit at all, but after reading it a few times I get what it does and why.

Reply:
Well:

1. we join the keys of `mappingMimetypeSubtype` with a `OR`
1. we try to match the string `subtype` and the joined string
1. we grab the first item of the result as it contains the matching substring

That is almost straightforward but it uses unusual JavaScript functions.